### PR TITLE
added lucide icons to be buttons to add h1,h2,h3

### PR DESCRIPTION
**Pull request by jasjs1**

### CODE REVIEW

File: html/buttons.html [hunk 1]
```diff
@@ -3,15 +3,20 @@
...
+  <img src="/assets/svg/buttonAssets/h2.svg" onclick="addH2()">
+  <img src="/assets/svg/buttonAssets/h3.svg" onclick="addH3()">
+
+  <br>
   <textarea id="markdown-editor" placeholder="Add or paste Markdown code here."></textarea>
   <div id="preview"></div>
   
 </body>
 <script src="/scripts/createMarkdownFormatting.js"></script>
+<script src="/scripts/addMarkdownFormattingWithButtons.js"></script>
```
Comment (Shoshta73) [inline html/buttons.html:21]: Already existing script?

Comment (jasjs1) [reply html/buttons.html:21]: That has not been put into production yet. I will create a PR for this.

Comment (jasjs1) [reply html/buttons.html:21]: > Already existing script?

I just added a pr for it

**Comment by jasjs1** [comment]
Not up to date.

**Pull request closed without merging**